README.md

```diff
@@ -97,12 +97,12 @@ Please :star: star :star: this project and help it grow! https://github.com/dgtl
 With Docker composer, just clone this repository and..
 
 ```bash
-$ docker-compose up -d
+docker compose up -d
```

Comment:
this is going to confuse people who run the older version (like me), better is to include both examples with what version each example applies for

Reply:
hi @dgtlmoon , I've add example and clarify for both version, would this work?